server/database/queries/cohort/putSpecificCohort.js

```diff
@@ -0,0 +1,7 @@
+const connection = require('../../config/connection');
+
+exports.putSpecificCohort = (id, req) =>
+  connection.query(
+    'UPDATE cohort SET name = $2, description = $3, img_url = $4, github_link = $5 WHERE id = $1 ',
```

Comment:
@Mu7ammadAbed I added a comment to another PR which might be helpful here too:
https://github.com/GSG-G8/ca-wiki/pull/47/files#r400924612

It describes how an ideal design for PUT requests is to have them handle requests that don't contain every single argument, so frontends can choose to PUT just a new name.